lbry/blockchain/sync/claims.py

```diff
@@ -281,3 +281,19 @@ def update_channel_stats(blocks: Tuple[int, int], initial_sync: int, p: Progress
     if result.rowcount and p.ctx.is_postgres:
         p.ctx.execute_notx(text("VACUUM claim;"))
     p.step(result.rowcount)
+
+
+@event_emitter("blockchain.sync.claims.filters", "claim_filters")
+def update_claim_filters(blocking_channel_hashes, filtering_channel_hashes, p: ProgressContext):
+    def select_reposts(channel_hashes, filter_type=0):
+        return select(
+            Claim.c.reposted_claim_hash, filter_type, Claim.c.channel_hash).where(
+            (Claim.c.channel_hash.in_(channel_hashes)) & (Claim.c.reposted_claim_hash.isnot(None)))
```

Comment:
If this query can be re-used in other places it should be moved out into the outer namespace. Also, the formatting should be something like:
```
        return (
            select(Claim.c.reposted_claim_hash, filter_type, Claim.c.channel_hash).where(
                (Claim.c.channel_hash.in_(channel_hashes)) &
                (Claim.c.reposted_claim_hash.isnot(None))
            )
        )
```